doc/pages/administrator/060-setup_cluster.md

```diff
@@ -8,7 +8,7 @@ Once the Astarte Operator [has been installed](030-installation_kubernetes.html)
 The standard way of deploying an Astarte instance is by creating your own Astarte Custom Resource.
 This gives you an high degree of customization, allowing you to tweak any single parameter in the
 Astarte setup. The main Astarte CRD contains [extensive
-documentation](https://github.com/astarte-platform/astarte-kubernetes-operator/blob/v1.0.1/config/crd/bases/api.astarte-platform.org_astartes.yaml)
+documentation](https://github.com/astarte-platform/astarte-kubernetes-operator/blob/v1.0.2/config/crd/bases/api.astarte-platform.org_astartes.yaml)
 on the available fields in OpenAPIv3 format. Just create your Astarte Custom Resource, which will
```

Comment:
Now we could refer to the new [CRD docs](https://docs.astarte-platform.org/snapshot/astarte-kubernetes-operator/)